docs/content/dev/public_api.yml

```diff
@@ -900,19 +900,21 @@ components:
     MediaUpload:
       type: object
       properties:
-        note:
-          type: string
-          description: ID of the note the file was uploaded to
-        user:
-          type: string
-          description: username of the user who uploaded the file
-        url:
+        link:
```

Comment:
Why the change from `url` to `link`?
I find "url" to be a way more descriptive term than "link".